anvil/geometry.py

```diff
@@ -190,12 +188,16 @@ def get_shift(self, shifts: tuple = (1, 1), dims: tuple = (0, 1)) -> torch.Tenso
     def two_point_iterator(self):
         """Generator which yields all the lattice shifts as one-dimensional tensors.
 
+        Yields
+        ------
+        torch.Tensor
+            one-dimensional tensor containing the shift indices for each lattice shift.
+
         Notes
         -----
         The order in which the shifts are generated is defined by the lexicographical
-        order of the Cartesian product of one-dimensional shifts. See the documentation
-        for itertools.product for details.
+        order of the Cartesian product of one-dimensional shifts. See :py:func:`itertools.product`.
         """
-        for shift_cart in product(range(self.length), range(self.length)):
+        for shift_cart in it.product(range(self.length), range(self.length)):
```

Comment:
corresponding change if you accept the above. Having said all that I think it was fine before.

```suggestion
        for shift_cart in itertools.product(range(self.length), range(self.length)):
```